src/EnergyPlus/UnitarySystem.cc

```diff
@@ -8234,6 +8235,11 @@ namespace UnitarySystems {
                 SingleDuct::SimATMixer(state, this->m_ATMixerName, FirstHVACIteration, this->m_ATMixerIndex);
             }
         }
+
+        // set variables used in fan call inside function calcPassiveSystem
+        state.dataUnitarySystems->m_massFlow1 = state.dataLoopNodes->Node(this->AirInNode).MassFlowRate;
+        state.dataUnitarySystems->m_runTimeFraction1 = (state.dataUnitarySystems->m_massFlow1 > 0.0) ? 1.0 : 0.0; // constant fan mode is required
+
```

Comment:
If either of these changes are removed, the system air flow goes to 0 and there is no control to set point temperature. I'm not quite sure the initialization of LastMode should be done in getInput but I am warming up to it since the mode needs to start somewhere. If the mode is not set from 0 to cooling or heating mode the fan does not turn on. So this change seems OK. And if the fan operating parameters are not set, for set point controlled systems, then the fan won't turn on. So this also seems OK.